### PR TITLE
Adding MODS profile

### PR DESCRIPTION
**Pull request by glenrobson**

### CODE REVIEW

File: source/api/registry/profiles/index.md [hunk 1]
```diff
@@ -50,10 +50,11 @@ The process for having a new entry added to this registry is [described here][re
 
 ## 3. Registry
 
-This table summarizes the known behaviors available, for use with the [Presentation API][prezi-api].
+This table summarizes the known profiles available, for use with the [Presentation API][prezi-api].
 
-| Behavior                       | 
+| Profile  | Description        | 
 | ------------------------------ |
+| http://www.loc.gov/mods/v3  | The URI for identifying [MODS (Metadata Object Description Schema)](http://www.loc.gov/standards/mods/index.html) metadata records.  |
```
Comment (zimeon) [inline source/api/registry/profiles/index.md:57]: I think the documentation link should be https: https://www.loc.gov/standards/mods/mods-schemas.html

**Review by zimeon — CHANGES_REQUESTED** [review]
Good by me once the MODS documentation link is fixed

**Review by jpstroop — DISMISSED** [review]

**Comment by glenrobson** [comment]
Thanks, I've updated the MODS link.

**Review by azaroth42 — APPROVED** [review]

**Review by mcwhitaker — DISMISSED** [review]

File: source/api/registry/profiles/index.md [hunk 2]
```diff
@@ -50,10 +50,11 @@ The process for having a new entry added to this registry is [described here][re
 
 ## 3. Registry
 
-This table summarizes the known behaviors available, for use with the [Presentation API][prezi-api].
+This table summarizes the known profiles available, for use with the [Presentation API][prezi-api].
 
-| Behavior                       | 
+| Profile  | Description        | 
 | ------------------------------ |
+| http://www.loc.gov/mods/v3  | The URI for identifying [MODS (Metadata Object Description Schema)](https://www.loc.gov/standards/mods/) metadata records.  |
```
Comment (zimeon) [inline source/api/registry/profiles/index.md:57]: I think the human text should specify v3

**Review by zimeon — CHANGES_REQUESTED** [review]

File: source/api/registry/profiles/index.md (hunk 2 above)
Comment (glenrobson) [reply source/api/registry/profiles/index.md]: Thanks I've added that in the human readable text.

File: source/api/registry/profiles/index.md (hunk 1 above)
Comment (glenrobson) [reply source/api/registry/profiles/index.md]: I've made that change.

**Review by azaroth42 — APPROVED** [review]

**Review by mikeapp — APPROVED** [review]

**Review by tomcrane — APPROVED** [review]

**Review by kirschbombe — APPROVED** [review]

**Review by zimeon — APPROVED** [review]

**Review by mixterj — APPROVED** [review]

**Pull request merged**